great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -330,6 +331,11 @@ def _check_input_validation(
                     "passed": False,
                 }
             )
+        if "validate_configuration" not in expectation_instance.__class__.__dict__:
+            passed = False
+            sub_messages.append(
+                {"message": "No validate_configuration method defined", "passed": False}
+            )
         else:
```

Comment:
This else clause was to the try/except block above. There are instances where the `first_test` variable will not exist (when `examples` is empty). Can you re-add the else block to the try/except and then move this if/else under that?